src/js/components/Breadcrumb.js

```diff
@@ -242,18 +232,23 @@ class Breadcrumb extends React.Component {
       // Decode Label
       let label = decodeURIComponent(pathParts[pathParts.length - 1]);
       let labelIsParam = label.charAt(0) === ':' && label.slice(1) in params;
+      if (labelIsParam) {
+        // Remove colon
+        label = label.slice(1);
+      }
 
       let crumbObjects = buildCrumb(
         label,
         name,
         params,
-        labelIsParam
+        labelIsParam,
+        buildCrumbCustomParams
```

Comment:
Oops, totally forgot about this. This is what happens when I try to build a theoretical component instead of implementing it in parallel (':